src/test/java/com/amihaiemil/docker/mock/PayloadOf.java

```diff
@@ -0,0 +1,169 @@
+package com.amihaiemil.docker.mock;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Map;
+import java.util.Set;
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonNumber;
+import javax.json.JsonObject;
+import javax.json.JsonString;
+import javax.json.JsonValue;
+import org.apache.http.HttpEntityEnclosingRequest;
+import org.apache.http.HttpRequest;
+
+/**
+ * Encapsulates the JSON payload of an HttpRequest as a JSON object.
```

Comment:
@llorllale I would say simply ``JSON payload of an HttpRequest``

Reply:
@amihaiemil ok